### PR TITLE
Chore: Add PHP tests

### PR DESCRIPTION
**Pull request by JeanPaiva**
Add unit tests to the plugin.

### CODE REVIEW

File: .env.testing-sample
Comment (tomusborne) [inline .env.testing-sample:1]: Does this mean anyone testing has to set up this same environment?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -139,7 +139,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: ['5.6', '7.4', '8.1']
```
Comment (tomusborne) [inline .github/workflows/ci.yml:142]: No more 5.6 testing?

File: .gitignore
Comment (tomusborne) [inline .gitignore:1]: We'll need to update `.distignore` with all newly-added files/folders as well.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (JeanPaiva) [reply .github/workflows/ci.yml:142]: Yes, the tests require at least PHP 7.2

**Review by tomusborne — APPROVED** [review]

**Pull request merged**